xapian-core/include/xapian/stem.h

```diff
@@ -30,7 +30,9 @@
 #include <xapian/intrusive_ptr.h>
 #include <xapian/visibility.h>
 
-#include <string>
+#include <iostream>
```

Comment:
Nothing in the library (and especially nothing in the public API headers) should need to be including `<iostream>`.